packages/orbit-components/src/Toast/__tests__/index.test.jsx

```diff
@@ -1,12 +1,22 @@
 // @flow
 import * as React from "react";
-import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import { render, screen, fireEvent, act } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 
 import { ToastRoot, createToast } from "..";
 import { Airplane } from "../../icons";
 import Toast from "../ToastMessage";
 import Button from "../../Button";
+import { EXPIRE_DISMISS_DELAY, SWIPE_DISMISS_DELAY } from "../consts";
+
+beforeEach(() => {
+  // reset mocks before each test
+  jest.useFakeTimers();
+});
+
+afterAll(() => {
+  jest.useRealTimers();
```

Comment:
Official docs recommend to use `usePendingTimers` before `useRealTimers`. [Here](https://testing-library.com/docs/using-fake-timers/).
> It's important to also call runOnlyPendingTimers before switching to real timers. This will ensure you flush all the pending timers before you switch to real timers.